test/JDBC/input/test_drop_login-vu-prepare.sql

```diff
@@ -0,0 +1,17 @@
+-- Create a login with sysadmin privileges
+CREATE LOGIN babel_5127_test_sysadmin WITH PASSWORD = '12345678';
+GO
+
+ALTER SERVER ROLE sysadmin ADD MEMBER babel_5127_test_sysadmin;
+GO
+
+-- Create a login with securityadmin privileges
+CREATE LOGIN babel_5127_test_securityadmin WITH PASSWORD = '12345678';
+GO
+
+ALTER SERVER ROLE securityadmin ADD MEMBER babel_5127_test_securityadmin;
+GO
```

Comment:
Can you please merge this tests with any existing test file ? Otherwise changes looks good!